test/commands/provar/automation/metadata/download.nut.ts

```diff
@@ -1,194 +1,332 @@
-import * as fileSystem from 'node:fs';
-import { execCmd, TestSession } from '@salesforce/cli-plugins-testkit';
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import * as fileSystem from 'node:fs/promises';
+import * as path from 'node:path';
+import { execCmd } from '@salesforce/cli-plugins-testkit';
 import { expect } from 'chai';
-import { errorMessages, commandConstants, SfProvarCommandResult } from '@provartesting/provardx-plugins-utils';
+import {
+  errorMessages,
+  commandConstants,
+  SfProvarCommandResult,
+  setNestedProperty,
+} from '@provartesting/provardx-plugins-utils';
+import { Global } from '@salesforce/core';
 import * as validateConstants from '../../../../assertion/validateConstants.js';
 import * as metadataDownloadConstants from '../../../../assertion/metadataDownloadConstants.js';
 
 describe('sf provar config metadataDownload NUTs', () => {
-  let session: TestSession;
   const DOWNLOAD_ERROR = 'Error (1): [DOWNLOAD_ERROR]';
-  enum FILE_PATHS {
-    METADATA_ERROR_FILE = 'metadataErrorFile.json',
-    MALFORMED_METADATA_FILE = 'malformedMetadataFile.json',
-    METADATA_DOWNLOAD_FILE = 'metadataDownloadFile.json',
-  }
-
-  after(async () => {
-    await session?.clean();
-    Object.values(FILE_PATHS).forEach((filePath) => {
-      fileSystem.unlink(filePath, (err) => {
-        if (err) {
-          return err;
-        }
-      });
-    });
+  it('Update Provar Home and ProjectPath in provar dx properties file', async () => {
```

Comment:
it() is something that executes a scenario that is it is a single NUT in itself. So, If we are not executing any scenario here, and just setting up the pre-requisites for further NUTs, then we should have all this in before() method

Reply:
Added in before method